### ISSUE TITLE
How to set up the adversarial training?

### ISSUE THREAD

**Issue by Dreamkeeper66666** [AlignmentResearch/go_attack#141]
Hi, I just tried to have the cyclic-adv-s545 model play against the latest 28b model. However, it doesn't seem to work very well. So I would like to do some fine tuning. I just saw some scripts under `kubernetes` folder, but I don't really know how to make it run, so  is there any instructions on setting up the adversarial training?  By the way is there some more recent models? Thanks!

**Comment by tomtseng** [comment]
Right, I don't expect cyclic-adv-s545m to work well against 28b since KataGo has been adversarially trained against cyclic positions from cyclic-adv-s545m, though I would guess that if you spent several hundred GPU-days fine-tuning that you could get an attack against 28b.

I added some more detailed instructions in the [README](https://github.com/AlignmentResearch/go_attack/blob/main/README.md#launching-victim-play-training-runs) now and uploaded adversaries to [Google Drive](https://drive.google.com/drive/u/3/folders/1-bGX-NQOh6MuRPoXJgYHb9-jWRJvviSg). I should note the scale of these experiments is a bit high (several hundred GPU-days) so if you don't have access to several GPUs you might find it hard to make much training progress.
